spec/policies/organisation_policy_spec.rb

```diff
@@ -4,10 +4,11 @@
   subject { described_class }
 
   permissions :index? do
-    it "is forbidden to organisation admins and normal users" do
+    it "is forbidden to super org admins, organisation admins and normal users" do
```

Comment:
nitpick: `org` => `organisation` for consistency (I'd probably have done the same thing though).

Reply:
You might want to check the rest of the PR for similar `org` vs. `organisation` changes too.  FWIW: I think `org` is fine in variable names, but we should probably use `organisation` in test names.